test/utils/deriveKeypair/index.ts

```diff
@@ -0,0 +1,34 @@
+import assert from 'assert-diff'
+import {deriveKeypair} from 'ripple-keypairs/dist'
```

Comment:
We don't need to test ripple-keypairs code here, it's already tested there - this can be deleted